ProcessLib/ProcessOutput.cpp

```diff
@@ -12,6 +12,97 @@
 #include "MeshLib/IO/VtkIO/VtuInterface.h"
 #include "NumLib/DOF/LocalToGlobalIndexMap.h"
 
+static void addSecondaryVar(double const t,
```

Comment:
Maybe rename to `addSecondaryVariable` or `addSecondaryVariableNodes`.

Reply:
Renamed to `addSecondaryVariableNodes` and also the corresponding residual function to `addSecondaryVariableResiduals`---now nice and consistent ;)